src/account/operations/balance.rs

```diff
@@ -143,8 +174,19 @@ impl AccountHandle {
 
                                 // Add amount
                                 total_amount += amount;
+
                                 // Add storage deposit
-                                required_storage_deposit += output_data.output.rent_cost(&rent_structure);
+                                if output_data.output.is_basic() {
+                                    required_storage_deposit.basic += rent;
+                                    // Amount for basic outputs isn't added to total_rent_amount, since we can spend it
+                                    // without burning.
```

Comment:
Should we also add it if the output has native tokens as here? https://github.com/iotaledger/wallet.rs/pull/1612/files#diff-22aa72010988d33cf882f809863b6642515ef8df5cf339ed256377f5f7491194R106-R114